client-src/clients/SockJSClient.js

```diff
@@ -36,4 +34,6 @@ module.exports = class SockJSClient extends BaseClient {
       f(e.data);
     };
   }
-};
+}
+
+export default SockJSClient;
```

Comment:
We need load this clients using `import()`

Reply:
we can just do `require(..).default` ?

they are imported in es5 code